### PR TITLE
F27 devel make vlan gui more robust

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Comment by rvykydal** [comment]
jenkins, test this please

File: pyanaconda/ui/gui/spokes/network.py [hunk 1]
```diff
@@ -429,6 +436,25 @@ def add_connection_to_list(self, con):
...
+        the value as <PARENT_IFACE>.<VLAN_ID> - same as NetworkManager.
+        """
+        iface = connection.get_setting_connection().get_interface_name()
+        if not iface:
+            setting_vlan = connection.get_setting_vlan()
+            if setting_vlan:
+                vlanid = setting_vlan.get_id()
+                parent = setting_vlan.get_parent()
+                # if parent is specified by UUID
+                if len(parent) == 36:
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/network.py]: What if the parent will have name with 36 characters but it won't be UUID ? Could that happen?

File: pyanaconda/ui/gui/spokes/network.py [hunk 2]
```diff
@@ -1020,12 +1020,16 @@ def _refresh_slaves(self, dev_cfg):
 
     def _refresh_parent_vlanid(self, dev_cfg):
         if dev_cfg.get_device_type() == NM.DeviceType.VLAN:
+            vlanidstr = parent = ""
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/network.py]: Please rename this variable. Sounds to me like someones name :)

Comment (rvykydal) [reply pyanaconda/ui/gui/spokes/network.py]: hehe, I'll try

File: pyanaconda/ui/gui/spokes/network.py (hunk 1 above)
Comment (rvykydal) [reply pyanaconda/ui/gui/spokes/network.py]: No, it can be either UUID or network iface name and the name size is limited to 15 or 16 chars in kernel

Comment (jkonecny12) [reply pyanaconda/ui/gui/spokes/network.py]: ok, then it should be fine :)

**Comment by rvykydal** [comment]
I got rid of the vlanidstr variable completely.

**Pull request merged**